workshop/session_1/03_modules_and_packages.ipynb

```diff
@@ -23,15 +23,15 @@
     "\n",
     "### What are modules?\n",
     "\n",
-    "In programming a **module** is part of a piece of software contating code that has a specific function. When developing large projects breaking code into modules helps keep the code **readable** and **maintainable**. For example, if we were building a game, a specific module may handle physics, while another handles what is rendered on screen. \n",
+    "In programming, a **module** is a part of software that contains code for a specific function. When developing large projects, breaking code into modules helps keep it **readable** and **maintainable**. For example, if we were building a game, one module might handle physics, while another manages what is rendered on the screen.\n",
```

Comment:
`software that contains code for a specific function` - this might be confusing in the context of function being a programming construct